wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/DeviceDetailsViewActionsHandler.swift

```diff
@@ -80,23 +81,30 @@ final class DeviceDetailsViewActionsHandler: DeviceDetailsViewActions, Observabl
         return nil
     }
 
+    @MainActor
     func removeDevice() async -> Bool {
-        isProcessing?(true)
         return await withCheckedContinuation {[weak self] continuation in
             guard let self = self else {
                 return
             }
+            self.continuation = continuation
```

Comment:
I don't really understand why it is requited to store the `continuation` object since it's only used within this closure?

Reply:
I will check this again.

Reply:
Calling continuation in any other way results in crash with "continuation is being reused more than once error" when one tries to cancel removal of the device upon display of a pop up that is presented upon tapping remove device.

Reply:
ok thanks